analyzer/codechecker_analyzer/cmd/check.py

```diff
@@ -96,6 +98,14 @@ def get_argparser_ctor_args():
 hash will not be changed so easily for example on code indentation or when a
 checker is renamed.
 
+Exit status
+------------------------------------------------
+0 - Successful analysis
```

Comment:
Should we mention here that successful analysis without any reports or any new reports?